pypeclub/OpenPype
# Nuke: fix multiple baking profile farm publishing

## Brief description
Multiple baking profiles were not correctly published via farm

## Description
Deadline submitter plugin didn't correctly formed expected files into json metadata. There were missing data for integrators to published correctly data to pipeline and ftrack. 

## Testing notes:
1. Open your favorite testing workfile
2. add multiple baking mov profiles - ideally one `withLut` and other `noLut` profiles. Second profile disable input process baking. But you have to have set input process viewer node activated. 
3. submit to farm and see there will be two published ftrack components one with baked look (input process) other without
